include/revng/TypeNames/PTMLCTypeBuilder.h

```diff
@@ -747,6 +747,23 @@ class CTypeBuilder : public CBuilder {
     return ptml::commentWithoutLeadingNewline(*this, T, "///", 0, Width);
   }
 
+  std::string
+  getWrapperStructComment(const model::RawFunctionDefinition &Function) const {
+    const model::Configuration &Configuration = Binary.Configuration();
+    uint64_t LineWidth = Configuration.commentLineWidth();
+
+    // TODO: do not rely on `Out`'s indentation, since there's no guarantee it's
+    //       the same stream (even if it usually is).
```

Comment:
This is spreading in more and more places and it's getting more cargo-cult.

Can't we get rid of this? And at least please centralize the computation in a single helper function, so that whenever we get down to fixing this there's just a single place to track down.

Reply:
Sounds good on the wrapper

Reply:
Done, please take a look at the `PTMLCBuilder: factor out line length calculation` commit:
```cpp
  uint64_t currentLineWidth() const {
    // TODO: Come up with something more robust here, because:
    //       1. there is no guarantee that the string is going to be written to
    //          the `Out` stream (this is only relevant for the methods that
    //          return strings).
    //       2. `currentIndentation()` does not take into account the fact that
    //          `Out` could be wrapping another indented stream.
    return Out->currentIndentation();
  }
```

Reply:
Ok, better than before, but I'd go even further and factor out all the uses of `currentLineWidth`, because they're all the same:
```c++
 uint64_t TotalLineWidth = Configuration.commentLineWidth();
 uint64_t Width = TotalLineWidth - currentLineWidth();
 ```
 
 These can be wrapped into a method.
 
Unless you foresee situations where `currentLineWidth()` and `commentLineWidth()` are used separately, but I would find that very suspicious.

Reply:
Done.